src/freenet/clients/http/QueueToadlet.java

```diff
@@ -1187,18 +1187,25 @@ else if(plainText != null)
 
 	}
 
-	protected String makeFetchKeysList(ClientContext context) throws PersistenceDisabledException {
+	protected String makeKeysList(ClientContext context, boolean inserts) throws PersistenceDisabledException {
 		RequestStatus[] reqs = fcp.getGlobalRequests();
 
-		StringBuffer sb = new StringBuffer();
+		StringBuilder sb = new StringBuilder();
 
 		for(RequestStatus req: reqs) {
-			if(req instanceof DownloadRequestStatus) {
+			if(!inserts && req instanceof DownloadRequestStatus) {
 				DownloadRequestStatus get = (DownloadRequestStatus)req;
 				FreenetURI uri = get.getURI();
 				sb.append(uri.toString());
 				sb.append("\n");
-			}
+			} else if (inserts && req instanceof UploadRequestStatus) {
+                UploadRequestStatus put = (UploadRequestStatus)req;
+				FreenetURI uri = put.getURI();
```

Comment:
Mixed style definitions... it will be shunted into uniformity during the bulk transformation of code that's likely to happen after 1468 is out and some other branches are merged.